kiwi/iso_tools/cdrtools.py

```diff
@@ -250,7 +250,8 @@ def _create_sortfile(self):
             boot_files = list(os.walk(self.source_dir + '/' + self.boot_path))
             boot_files += list(os.walk(self.source_dir + '/EFI'))
             for basedir, dirnames, filenames in boot_files:
-                for filename in filenames:
+                #dirnames.sort() # TODO - seems to break mocking
```

Comment:
left this for now. One workaround is to write it as `dirnames=sorted(dirnames)`
but IMHO fixing the tests would be better